### ISSUE TITLE
CDN broken

### ISSUE THREAD

**Issue by janhesters** [ethereum/web3.js#3074]
### Description

I tried to include Web3 using a CDN:

```html
<script src="https://cdn.jsdelivr.net/npm/web3@1.2.1/src/index.min.js"></script>
```

#### Expected behavior

Web3 is available in the browser.

#### Actual behavior

It breaks.

```js
Uncaught ReferenceError: require is not defined
    at index.js:31
```

#### Alternatives you tried

I tried loading it directly from GitHub (using `"https://cdn.jsdelivr.net/gh/ethereum/web3.js@1.2.1/dist/web3.min.js",`), but this breaks complaining about generators.

```js
Uncaught ReferenceError: regeneratorRuntime is not defined
    at Object.<anonymous> (web3.min.js:1)
    at Object.380.buffer (web3.min.js:1)
    at c (web3.min.js:1)
    at web3.min.js:1
    at Object.379../utils (web3.min.js:1)
    at c (web3.min.js:1)
    at web3.min.js:1
    at Object.377../scrypt (web3.min.js:1)
    at c (web3.min.js:1)
    at web3.min.js:1
```

### Versions 
* web3.js: 1.2.1
* nodejs: -
* browser: Chrome Version 77.0.3865.75 (Official Build) (64-bit)
* ethereum node: -

**Comment by yuzhoumao** [comment]
@janhesters it looks like the [landing page documentation](https://github.com/ethereum/web3.js#requirements) already lists the requirements of this library, that is,
> ### Requirements
>
> * [Node.js](https://nodejs.org)
> * npm

<del>so I don't think you can just include the cdn js file and expect it to work</del>
according to the documentation for installation, you can include the pure js file, https://web3js.readthedocs.io/en/v1.2.0/getting-started.html#adding-web3, but it looks to me that you are including the wrong file. should be using `dist/web3.min.js`

**Comment by janhesters** [comment]
@yuzhoumao Thank you for your help! I was able to include Web3 differently.

**Comment by yuzhoumao** [comment]
@janhesters You are welcome. Closing this issue.

**Comment by ethershare** [comment]
> @yuzhoumao Thank you for your help! I was able to include Web3 differently.

Hi， I have met the same problem. How do you fix this? Could you please kindly tell me

**Comment by jmlon** [comment]
Having the same problem, using the 'dist'  version form jsdelivr. Try both, firefox and chrome.

How it was included:
`<script src="https://cdn.jsdelivr.net/gh/ethereum/web3.js@1.2.1/dist/web3.min.js"></script>`

Error in chrome:

```
web3.min.js:1 Uncaught ReferenceError: regeneratorRuntime is not defined
    at Object.<anonymous> (web3.min.js:1)
    at Object.380.buffer (web3.min.js:1)
    at c (web3.min.js:1)
    at web3.min.js:1
    at Object.379../utils (web3.min.js:1)
    at c (web3.min.js:1)
    at web3.min.js:1
    at Object.377../scrypt (web3.min.js:1)
    at c (web3.min.js:1)
    at web3.min.js:1

```
Error in firefox:
```
ReferenceError: regeneratorRuntime is not definedweb3.min.js:1:606327
    [380]</< https://cdn.jsdelivr.net/gh/ethereum/web3.js@1.2.1/dist/web3.min.js:1
    [380]< https://cdn.jsdelivr.net/gh/ethereum/web3.js@1.2.1/dist/web3.min.js:1
    c https://cdn.jsdelivr.net/gh/ethereum/web3.js@1.2.1/dist/web3.min.js:1
    c https://cdn.jsdelivr.net/gh/ethereum/web3.js@1.2.1/dist/web3.min.js:1
    [379]< https://cdn.jsdelivr.net/gh/ethereum/web3.js@1.2.1/dist/web3.min.js:1
    c https://cdn.jsdelivr.net/gh/ethereum/web3.js@1.2.1/dist/web3.min.js:1
    c https://cdn.jsdelivr.net/gh/ethereum/web3.js@1.2.1/dist/web3.min.js:1
    [377]< https://cdn.jsdelivr.net/gh/ethereum/web3.js@1.2.1/dist/web3.min.js:1
    c https://cdn.jsdelivr.net/gh/ethereum/web3.js@1.2.1/dist/web3.min.js:1
    c https://cdn.jsdelivr.net/gh/ethereum/web3.js@1.2.1/dist/web3.min.js:1
    [397]</< https://cdn.jsdelivr.net/gh/ethereum/web3.js@1.2.1/dist/web3.min.js:1
    [397]< https://cdn.jsdelivr.net/gh/ethereum/web3.js@1.2.1/dist/web3.min.js:1
    c https://cdn.jsdelivr.net/gh/ethereum/web3.js@1.2.1/dist/web3.min.js:1
    c https://cdn.jsdelivr.net/gh/ethereum/web3.js@1.2.1/dist/web3.min.js:1
    [396]</< https://cdn.jsdelivr.net/gh/ethereum/web3.js@1.2.1/dist/web3.min.js:1
    [396]< https://cdn.jsdelivr.net/gh/ethereum/web3.js@1.2.1/dist/web3.min.js:1
    c https://cdn.jsdelivr.net/gh/ethereum/web3.js@1.2.1/dist/web3.min.js:1
    c https://cdn.jsdelivr.net/gh/ethereum/web3.js@1.2.1/dist/web3.min.js:1
    [417]< https://cdn.jsdelivr.net/gh/ethereum/web3.js@1.2.1/dist/web3.min.js:1
    c https://cdn.jsdelivr.net/gh/ethereum/web3.js@1.2.1/dist/web3.min.js:1
    c https://cdn.jsdelivr.net/gh/ethereum/web3.js@1.2.1/dist/web3.min.js:1
    Web3 https://cdn.jsdelivr.net/gh/ethereum/web3.js@1.2.1/dist/web3.min.js:1
    c https://cdn.jsdelivr.net/gh/ethereum/web3.js@1.2.1/dist/web3.min.js:1
    o https://cdn.jsdelivr.net/gh/ethereum/web3.js@1.2.1/dist/web3.min.js:1
    <anonymous> https://cdn.jsdelivr.net/gh/ethereum/web3.js@1.2.1/dist/web3.min.js:1
    <anonymous> https://cdn.jsdelivr.net/gh/ethereum/web3.js@1.2.1/dist/web3.min.js:1
    <anonymous> https://cdn.jsdelivr.net/gh/ethereum/web3.js@1.2.1/dist/web3.min.js:1

```

**Comment by KazekoToHaloers** [comment]
I have the same problem with the @jmlon 
``````````
Uncaught ReferenceError: regeneratorRuntime is not defined        web3.min.js:1
    at Object.<anonymous> (web3.min.js:1)
    at Object.380.buffer (web3.min.js:1)
    at c (web3.min.js:1)
    at web3.min.js:1
    at Object.379../utils (web3.min.js:1)
    at c (web3.min.js:1)
    at web3.min.js:1
    at Object.377../scrypt (web3.min.js:1)
    at c (web3.min.js:1)
    at web3.min.js:1

**Comment by nivida** [comment]
This will be fixed with publishing of version 1.2.2. The minified file located in the dist folder of the 1.x branch does already have this fix applied.

**Comment by cgewecke** [comment]
We now test every PR by publishing a simple site that uses web3.min.js at:

http://sudden-playground.surge.sh/

The CDN should work without issue, please ping if anyone's still seeing this.
